content/push/index.textile

```diff
@@ -0,0 +1,81 @@
+---
+title: Push notifications
+meta_description: "Ably delivers push notifications to user devices."
+meta_keywords: "Push, push notifications, Apple push notification service, Google Firebase cloud messaging service, APNs, FMC"
+redirect_from:
+  - /general/push
+  - /rest/push
+  - /rest/versions/v1.1/push
+  - /realtime/push
+  - /realtime/versions/v1.1/push
+  - /general/versions/v1.1/push
+  - /general/versions/v1.0/push
+  - /general/push-notifications
+  - /general/versions/v0.8/push-notifications
+  - /general/smart-notifications
+---
+
```

Comment:
This reads like it implies you can't receive notifications whilst the app is open/in the foreground

Reply:
good point

Reply:
Push notifications notify user devices whether or not an application is open and running.